flow/designs/intel22/aes/constraint.sdc

```diff
@@ -1,2 +1,5 @@
-create_clock [get_ports clk] -period 5000
+set clk_name clk
+set clk_period 2600
+#
+create_clock -name $clk_name -period $clk_period [get_ports clk]
```

Comment:
Add IO constraints like you did with the other designs.

Reply:
Updated all SDCs to have IO constraints.